site/src/components/site-header.md

```diff
@@ -0,0 +1,499 @@
+---
+title: Site header
+description: The Site header component in the Atlas Design System
+template: standard
+classType: Component
+classPrefixes:
+  - site-header
+---
+
+# Site header
+
+Site header component allows adding full width container to hold everything site's header might need - the logo, brand link, navigation, and other controls.
+
+**This page is best viewed with the content section full screened because banners are intended to be full width elements. Go full screen by clicking the button in the top right corner.**
+
+## Usage
+
+The site header is composed of the following elements:
+
+| Component             | Description                                                 |
+| --------------------- | ----------------------------------------------------------- |
+| `site-header`         | The main container for the site header.                     |
+| `site-header-button`  | Buttons within the site header.                             |
+| `site-header-logo`    | The logo displayed in the site header.                      |
+| `site-header-divider` | A vertical divider between elements.                        |
+| `site-header-brand`   | The brand name or title in the site header.                 |
+| `site-header-nav`     | The navigation container within the site header.            |
+| `site-header-panel`   | A panel that can be expanded to display additional content. |
+
+Below is the example of the site header with logo, brand and navigation links.
```

Comment:
I am an unsufferable boor 😔, but I'm sorry I have to suggest the Oxford comma ...

```suggestion
Below is the example of the site header with logo, brand, and navigation links.
```